packages/vscode-extension/package.json

```
@@ -175,7 +192,31 @@
           "type": "webview",
           "id": "RadonIDE.view",
           "name": "",
-          "when": "config.RadonIDE.panelLocation != 'tab' && !RNIDE.sidePanelIsClosed"
+          "when": "RNIDE.extensionIsActive && config.RadonIDE.panelLocation != 'tab' && !RNIDE.sidePanelIsClosed"
+        }
+      ],
+      "RadonIDEToolExpoDevPluginReactQuery": [
+        {
+          "type": "webview",
+          "id": "RNIDE.Tool.ExpoDevPlugin.ReactQuery.view",
+          "name": "",
+          "when": "RNIDE.Tool.ExpoDevPlugin.ReactQuery.available"
```

```suggestion
          "when": "RNIDE.Tool.ExpoDevPlugin.ReactQuery.available && RNIDE.panelIsOpen"
```

I think we should hide plug in panels when no app is running